scripts/ci/libraries/_initialization.sh

```diff
@@ -726,12 +728,8 @@ function initialization::get_environment_for_builds_on_ci() {
         export CI_REF="${CI_REF="refs/head/master"}"
     fi
 
-    if [[ ${VERBOSE} == "true" && ${PRINT_INFO_FROM_SCRIPTS} == "true" ]]; then
-        initialization::summarize_build_environment
-    fi
-
     if [[ -z "${LIBRARY_PATH:-}" && -n "${LD_LIBRARY_PATH:-}" ]]; then
-      export LIBRARY_PATH="$LD_LIBRARY_PATH"
+      export LIBRARY_PATH="${LD_LIBRARY_PATH}"
```

Comment:
Why change this line?

Reply:
This is something I learned to always use braces for variables. While they are optional, they give clear boundaries. So I add them by default for all variables always. I don't even think about it when I do whenever I am around such variable. 

It's also consistent with Google shell guide which we adopted: 

https://google.github.io/styleguide/shellguide.html#variable-expansion

> In order of precedence: Stay consistent with what you find; quote your variables; prefer "${var}" over "$var".

> Don’t brace-delimit single character shell specials / positional parameters, unless strictly necessary or avoiding deep confusion.
> Prefer brace-delimiting all other variables.

Reply:
It's more the change of this line in this PR I was commenting on. By including changes like this that are otherwise unrelated to the PR at hand it makes it much harder to review as there is more diff to look through.